persistence/registry/run/persist-via-run-registry-key.yml

```diff
@@ -9,20 +9,35 @@ rule:
     examples:
       - Practical Malware Analysis Lab 06-03.exe_:0x401130
       - b87e9dd18a5533a09d3e48a7a1efbcf6:0x1400070E0
+      - 9ff8e68343cc29c1036650fc153e69f7:0x470624
   features:
     - or:
       - and:
         - or:
-          - or:
-            - api: advapi32.RegOpenKey
-            - api: advapi32.RegOpenKeyEx
-          - or:
-            - api: advapi32.RegSetValue
-            - api: advapi32.RegSetValueEx
-          - or:
-            - number: 0x80000001 = HKEY_CURRENT_USER
-            - number: 0x80000002 = HKEY_LOCAL_MACHINE
-        - string: /Software\\Microsoft\\Windows\\CurrentVersion\\Run/i
+          - match: set registry value
+          - number: 0x80000001 = HKEY_CURRENT_USER
+          - number: 0x80000002 = HKEY_LOCAL_MACHINE
+        - or:
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\Run/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\RunOnce/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\RunOnceEx/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Shell Folders/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\User Shell Folders/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\RunServices/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\RunServicesOnce/i
+          - string: /Software\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explorer\\Run/i
+          - string: /Software\\Microsoft\\Windows NT\\CurrentVersion\\Windows\\load/i
+          - string: /System\\CurrentControlSet\\Control\\Session Manager\\BootExecute/i
```

Comment:
These now capture more than just the Run keys (which is consistent with the att&ck technique). Should we rename the rule to `persist via registry key` or similar?